### ISSUE TITLE
same idea?

### ISSUE THREAD

**Issue by Sundrops** [DefangChen/SimKD#8]
- proxylesskd direct knowledge distillation with inherited classifier for face recognition https://arxiv.org/pdf/2011.00265.pdf

**Comment by DefangChen** [comment]
Thanks for bringing this arxiv paper to our attention. Since I am unfamiliar with the Face Recognition field, I just took a glance over the first three sections to give you a quick response.
	
Several technical differences are listed as follows:
	
1. We aim to develop a representation-free knowledge distillation, and thus the teacher and student features are aligned explicitly without label information. In contrast, this arxiv paper trains the student model with labels only, without an explicit knowledge distillation loss.
	
2. Our paper adopts an extremely simple l2 loss for feature alignment in the position of pre-classifier. While this arxiv paper adopts a modified cross-entropy loss, such as Cosface, Arcface in the position of post-classifier. I guess this loss distinction stems from the preference for different tasks (Our paper considers a general classification task while this arxiv paper focuses on a fine-grained face recognition task).
	
3. Based on the above two differences, the implementation of multi-teacher knowledge distillation is also different.
	
4. Additionally, we provide an in-depth analysis of alternative strategies to deal with the training of student feature encoder and classifier. A projector is proposed to enable our technique applicable to arbitrary model architectures. Both are not presented in this arxiv paper.
	
Although I did not read this arixv paper before, I do treat this paper as a related (but not the same) one and appreciate its contribution after fast browsing. It would be beneficial to clearly highlight the difference in our paper.
